.github/workflows/ci.yml

```
@@ -86,7 +86,7 @@ jobs:
     needs: changes
     strategy:
       matrix:
-        target: [ubuntu-latest, macos-latest, windows-latest]
```

removed by accident?

---

I was getting path too long errors when cargo was cloning it, and since CI was fail fast, i couldn't actually tell if it was building/working correctly